backend/supervisor.d/services.conf

```diff
@@ -134,7 +134,7 @@ directory=/root/core
 umask=022
 autostart=false
 autorestart=unexpected
-startsecs=10
+startsecs=1
```

Comment:
the waiting time is only needed in the testing configuration where we want to avoid any race condition while not sleeping